k8sfv/leastsquares/least_squares_method.go

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) 2017 Tigera, Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Code copied from https://gist.github.com/ota42y/db4ff0298d9c945cd261, with some modifications.
+
+package leastsquares
+
+type Point struct {
+	X float64
+	Y float64
+}
+
+func LeastSquaresMethod(points *[]Point) (a float64, b float64) {
```

Comment:
I don't think you need to pass points by pointer, a slice is already a pointer-ish type.